test/tests/functional/pbs_resv_end_hook_set_attrs.py

```diff
@@ -0,0 +1,61 @@
+from tests.functional import *
+import time
+
+
+class TestSetAttributes(TestFunctional):
```

Comment:
All these test plans are about the resv_end hook.  Don't create a bunch of little test plans.  Create one larger one.